website/docs/cli/inspect/index.html.md

```diff
@@ -1,6 +1,7 @@
 ---
 layout: "docs"
 page_title: "Inspecting Infrastructure - Terraform CLI"
+description: "Learn commands to inspect and transform dependency information, outputs, etc. Use them for integration or to more deeply understand your infrastructure."
```

Comment:
even though the content says "Terraform CLI includes some commands for inspecting or transforming this data.". All the commands on this page don't actually directly modify the data. The closest thing is the `graph` command, but that takes the information and returns it in DOT format -- the existing information is unaltered

Reply:
Thank you this is super helpful. I'm going to push up a change that omits the transform language